src/main/java/thePackmaster/powers/contentcreatorpack/RetromationPower.java

```diff
@@ -26,38 +27,30 @@ public class RetromationPower extends AbstractPackmasterPower {
     public static final String NAME = powerStrings.NAME;
     public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
 
-    private ArrayList<AbstractCardPack> potentialPacks = new ArrayList<>();
+    private List<AbstractCard> potentialCards = new ArrayList<>();
 
     public RetromationPower(int amount) {
         super(POWER_ID, NAME, PowerType.BUFF, false, AbstractDungeon.player, amount);
-        if (SpireAnniversary5Mod.allPacksMode) {
-
-            //If in ALL PACKS mode, then this has to naturally just include everything.
-            potentialPacks = allPacks;
+        potentialCards = this.getPotentialCards();
+    }
 
-        } else {
-            for (AbstractCardPack p : allPacks) {
-                if (!SpireAnniversary5Mod.currentPoolPacks.contains(p) && !(p instanceof GemsPack)) {
-                    potentialPacks.add(p);
-                }
-            }
-        }
+    private List<AbstractCard> getPotentialCards() {
+        return CardLibrary.getAllCards().stream()
```

Comment:
While Mimicry, More Explosions, and Favor all create cards from your current card pool, Retromation makes cards from outside your current card pool. Specifically it says "At the start of your turn, gain a card from a Pack not available in your draft."

If you're not playing Packmaster, there are no packs in your draft, so having this make cards from all Packmaster cards seems the most sensible. The code for that isn't as easy as for the other cards, but it's still a bit simpler than it was before.